docs/excel/excel-add-ins-worksheet-display.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Adjust worksheet display settings
+description: How to adjust some worksheet display settings to make reports easier to read.
+ms.date: 04/03/2025
+ms.localizationpriority: medium
+---
+
+# Adjust worksheet display settings
+
+Excel is often used for reporting scenarios where you want to share worksheet data with others. Your Office Add-in can reduce visual clutter and help focus attention by controlling the appearance of the worksheet. The Office JavaScript API supports changing several visual aspects of the worksheet.
+
+## Page layout and print settings
+
+Add-ins have access to page layout settings at a worksheet level. These control how the sheet is printed. A `Worksheet` object has three layout-related properties: `horizontalPageBreaks`, `verticalPageBreaks`, `pageLayout`.
```

Comment:
Should we add something about PDF vs. physical printing? Or is it implied that this applies to PDFs too? 

```suggestion
Add-ins have access to page layout settings at a worksheet level. These control how the sheet is printed. A `Worksheet` object has three layout-related properties: `horizontalPageBreaks`, `verticalPageBreaks`, and `pageLayout`.
```

Reply:
For the purposes of the APIs, printing to a PDF and a physical printer should be the same.